packages/router/src/page-loader.tsx

```diff
@@ -101,6 +101,8 @@ export class PageLoader extends React.Component<Props> {
       slowModuleImport: false,
       params: props.params,
     })
+
+    window.scrollTo(0, 0)
```

Comment:
```suggestion
    window?.scrollTo(0, 0)
```

Reply:
That won't work I'm afraid! It'll still throw

Reply:
Why/how? I thought `window` would just be `undefined`

Reply:
Try this in your browser console. 

`console.log(bazinga?.notThere)`

It throws because the safe access operator needs atleast the object to be defined.

Reply:
I see. I thought `window` would actually be `=== undefined`, so `console.log(undefined?.notThere)`. But now I understand that it's not even available at all.

Reply:
Hmmm... this is a good point, maybe I can shim window to be undefined in the prerender process, so that we can use this clean syntax!

Reply:
@dac09 Is this the way that you think we'll go with?

Reply:
I think I'd rather have a linter rule to be honest, because "monkey patching" window, even if its to undefined - makes me a little worried